src/Stratis.Features.FederatedPeg/FederatedPegOptions.cs

```diff
@@ -13,9 +13,15 @@ public class FederatedPegOptions
         /// </summary>
         public Network CounterChainNetwork { get; }
 
-        public FederatedPegOptions(Network counterChainNetwork)
+        /// <summary>
+        /// The height to start syncing the wallet from.
+        /// </summary>
+        public int WalletSyncFromHeight { get; }
+
+        public FederatedPegOptions(Network counterChainNetwork, int walletSyncFromHeight = 1)
```

Comment:
previously we kept `nodeSettings.Network.Consensus.MaxReorgLength + 1` I think there was a reason.

Reply:
Can you elaborate?